src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java

```diff
@@ -39,26 +40,41 @@
   FOREIGN_OFFICES("FOREIGN OFFICES", AddressType.HH),
   CASTLES("CASTLES", AddressType.HH),
   GRT_SITE("GRT SITE", AddressType.HH),
-  MILITARY_SFA("MILITARY SFA", AddressType.SPG),
-  EMBASSY("EMBASSY", AddressType.SPG),
-  ROYAL_HOUSEHOLD("ROYAL HOUSEHOLD", AddressType.SPG),
+  MILITARY_SFA("MILITARY SFA", AddressType.SPG, SecurityType.SECURE),
+  EMBASSY("EMBASSY", AddressType.SPG, SecurityType.SECURE),
+  ROYAL_HOUSEHOLD("ROYAL HOUSEHOLD", AddressType.SPG, SecurityType.SECURE),
   CARAVAN_SITE("CARAVAN SITE", AddressType.SPG),
   MARINA("MARINA", AddressType.SPG),
   TRAVELLING_PERSONS("TRAVELLING PERSONS", AddressType.SPG),
   TRANSIENT_PERSONS("TRANSIENT PERSONS", AddressType.SPG);
 
+  private static enum SecurityType {
+    SECURE,
+    NOT_SECURE
+  }
+
   private String code;
   private AddressType addressType;
+  private SecurityType securityType;
 
-  private EstabType(String code, AddressType addressType) {
+  private EstabType(String code, AddressType addressType, SecurityType securityType) {
     this.code = code;
     this.addressType = addressType;
+    this.securityType = securityType;
+  }
+
+  private EstabType(String code, AddressType addressType) {
```

Comment:
Would rather not overload the enum constructor, instead call a spade a spade.
Thinking of the next dev as well who needs to add another field after SecurityType...

Reply:
I initially went for the simple approach. Every estab type specified its security type. Moved away from that as the table became a big splodge of text (especially when autoformatting means you can't line things up).
There are pros and cons to everything. Agree that you can't have more and more constructors if there are more attributes. However, there aren't more attributes at the moment. If more attributes are added then this may need to be revised. 
Happy to change if still want me to remove the new constructor.

Reply:
FWIW I'm perfectly happy with Pete's approach!

Reply:
FTR Having the explicit declaration in the construction makes it clear and unambiguous